resources/azepi/apply.py

```diff
@@ -51,6 +53,40 @@ def _run_epicli_apply(v):
             v["epiphany_file"].unlink()
 
 
+def _ensure_ssh_key_permissions(v):
+    """Apply SSH key permissions workaround for Docker on Windows."""
+
+    module_config = load_yaml(v["config_file"])[v["M_MODULE_SHORT"]]
+
+    epiphany_config_docs = load_yaml(module_config["config"])
+
+    cluster = select(epiphany_config_docs,
+                     q_kind("epiphany-cluster"),
+                     exactly=1)
+
+    key_path_from_spec = cluster["specification"]["admin_user"]["key_path"]
+
+    key_path_from_share = os.path.join(v["shared_dir"], "vms_rsa")
```

Comment:
Would be nice to keep the convention and use `pathlib.Path` here as well :+1::innocent:
```suggestion
    key_path_from_share = v["shared_dir"] / "vms_rsa"
```